### PR TITLE
Sum of Cubes of first N natural numbers

### LINKED ISSUE #1274: Sum of cubes of first N natural numbers
In C, C++, Java, Python and Go

### PR DESCRIPTION
**Pull request by singh-shreya6**
Fixes #1274 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- Added program for sum of cubes of first N natural numbers

#### Languages Used:
- C
- C++
- Java
- Python
- Go

#### Files Added:
- Sum_Of_Cubes.c
- Sum_Of_Cubes.cpp
- Sum_Of_Cubes.java
- Sum_Of_Cubes.py
- Sum_Of_Cubes.go

### CODE REVIEW

File: Sum_Of_Cubes/Sum_Of_Cubes.cpp [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    (N * (N + 1) / 2) ^ 2
+*/
+
+#include <bits/stdc++.h>
+using namespace std;
+
+int main()
+{
+    int N;
+    cout << "Enter the value of N: " << endl;
```
Comment (marwahmanbir) [inline Sum_Of_Cubes/Sum_Of_Cubes.cpp]: Please remove spaces around ``<<`` and ``>>`` in cout and cin statements as these are not required and the compiler simply ignores unnecessary spaces.

File: Sum_Of_Cubes/Sum_Of_Cubes.c [hunk 2]
```diff
@@ -0,0 +1,32 @@
+/*
+    The sum of cubes of first N natural numbers is given
+    by the equation (N * (N + 1) / 2) ^ 2 i.e.
+    1^3 + 2^3 + 3^3 + 4^3 + 5^3.........N^3 sums upto
+    (N * (N + 1) / 2) ^ 2
+*/
+
+#include <math.h>
+#include <stdio.h>
```
Comment (marwahmanbir) [inline Sum_Of_Cubes/Sum_Of_Cubes.c]: Remove space after ``include``.

File: Sum_Of_Cubes/Sum_Of_Cubes.cpp [hunk 3]
```diff
@@ -0,0 +1,31 @@
+/*
+    The sum of cubes of first N natural numbers is given
+    by the equation (N * (N + 1) / 2) ^ 2 i.e.
+    1^3 + 2^3 + 3^3 + 4^3 + 5^3.........N^3 sums upto
+    (N * (N + 1) / 2) ^ 2
+*/
+
+#include <bits/stdc++.h>
```
Comment (marwahmanbir) [inline Sum_Of_Cubes/Sum_Of_Cubes.cpp]: Remove space after ``include``.

**Comment by singh-shreya6** [comment]
Done @marwahmanbir . Please Review.

**Comment by dipu989** [comment]
@marwahmanbir  I have implemented the code using JavaScript for this issue.
Should I open a new issue and then Open a PR? 
Or can I open the PR referencing this issue only?

File: Sum_Of_Cubes/Sum_Of_Cubes.c [hunk 4]
```diff
@@ -0,0 +1,32 @@
+/*
+    The sum of cubes of first N natural numbers is given
```
Comment (marwahmanbir) [inline Sum_Of_Cubes/Sum_Of_Cubes.c]: Good work.
In future PRs, please do not indent the comments that you add on the top like this (descriptions).

**Pull request merged**